### DISCUSSION TITLE
Improving Ray's documentation about connecting with windows wsl2 instance

### DISCUSSION THREAD

**Discussion by pablosebastianr** [spatie/laravel-ray#235] [Ideas]
I was having issues with Ray not showing the data sent from the laravel application running on a Debian wsl instance

I checked and double checked everytyhing, local_path, remote_path, but problem was the host variable

My windows 10 host was 192.168.xxx.xxx and that was what I was setting in ray.php. My debian machine was able to ping that ip address. But no luck.
So I checked again with ipconfig /all and I saw another ip, the one I got from my vpn for work.... 10.54.xxx.xxx, so I tried that one for the host variable and that worked.

I think it would be nice if this was mentioned in https://spatie.be/docs/ray/v1/environment-specific-configuration/windows-10,  since it would save up some time dealing with a rather obscure issue

**Comment by freekmurze** [comment]
Feel free to add a concise paragraph about this in our docs, there's an edit button on top of the page 👍
